components/broadcaster_card/commons/broadcaster_card.lua

```diff
@@ -100,6 +101,9 @@ function BroadcasterCard.create(frame)
 			date = date,
 			sort = tonumber(args[prefix .. 'sort'])
 		}
+		--if entered name and stored name as well as entered flag and stored flag match mark it as manual input
+		broadcaster.isManualInput = args[prefix .. 'name'] == name and args[prefix .. 'flag' ] == nationality
```

Comment:
if determined name equals input name and determined flag equals input flag consider it manual input

Reply:
Having that flag stored in extradata will prevent unfixable loops of wrong data in LPDB.
Currently, if you enter wrong data on one page, other pages can request it which leads to a loop where if the input is removed/fixed, pages will still use bad data

Reply:
It is mostly about casters without pages, that are using lpdb_broadcaster to get data.

Reply:
Also in your example the module just gets `testy tester` as flag, which isn't valid one

Reply:
bflag gets ignored entirely

Reply:
Option A: there is no queried data, but there is manual input. Then we set the flag, this data can be used as source
Option B: there is queried data, but no manual input. Condition is false, queries of this data won't be taken as source on other pages
Option C: there is queried data and manual input, which matches: data can be used as source

Reply:
It will prevent occasions like this:
https://discord.com/channels/93055209017729024/268719633366777856/1106894550775120023
https://discord.com/channels/93055209017729024/268719633366777856/1106895358031822909

Reply:
you could have partial input like only name known
in that case it would still try to query broadcaster cards and come up with no flag via that either but it queried
and still is identical to manual input

Reply:
the reason why i did it indirectly is that in case of only 1 of flag/name known it will still try to query and hence the flag would never be set if done only if no query happens